### ISSUE TITLE
[remux] Support custom parameters for starting xochitl

### ISSUE THREAD

**Issue by ademasi** [rmkit-dev/rmkit#22]
Hi,

Thanks for your work, I really enjoy remux and the developing ecosystem that you are creating. I need to try your new library to create apps for the rm.

Anyway, I am using ddvk hack on xochitl to support the Lamy pen, to be able to have the eraser function by pushing the pen's button. 
Right now, with remux, it's not working because remux start it's own process and stop  xochitl via its systemd service : https://github.com/rmkit-dev/rmkit/blob/master/src/remux/launcher.cpy
```
def run():
   _ := system("systemctl stop xochitl")
  proc::launch_process("xochitl", true /* check running */, true /* background */)
```

[ddvk's hack start xochitl with](https://github.com/ddvk/remarkable-stylus/) a command line argument:
`xochitl -plugin evdevlamy` .

Would it be possible to include this feature to remux via a conf file ? 

I can write a push request  hardcoded for my own usage, but I like the fact that all your software is now available via opkg. Hence, if you have already defined an API for passing argument to apps in remux, I would be happy to use it and contribute to the software.

Regards, Alex.

**Comment by raisjnn** [comment]
there's currently no api for the described case, nor a config file for remux but welcome any steps that push us in that direction

this case in particular is special because it's xochitl, but i wonder if we can pull the correct binary invocation from the xochitl draft file or if we should just go ahead and make the remux config file

i leave it up to you on which direction to pursue

**Comment by raisjnn** [comment]
(btw: we did hardcode the plugin command in our local version of remux on the tablet that has remarkable hacks installed ;) )

**Comment by ademasi** [comment]
I hard coded it in my first try to test the toolchain to compile remux and the rmkit software. 

Currently, my implementation of the config file is stills in the design (and I am also learning okp, just to be sure, it's python in C syntax that generate C++  to be compiled to ARM binary? ).  The dot file of remux would be named ".remux" and place in the /home/root directory. Each parameter id would be inside brackets e.g. :
 
[remux]
-plugin evdevlamy

[calendar]
...
Thanks for the documentation of the project, it's super useful.

**Comment by raisjnn** [comment]
I'm glad the docs are helping, it makes me feel that the hours writing them was not wasted :-D

re: config file: it should be in `/home/root/.config/remux/remux.conf` (the newer style of config files on linux), i think that this particular parameter could make sense like below (or however .ini files specify key=value pairs):

```
[remux]
xochitl_flags=-plugin evdevlamy
```

re: cpy: CPY is a preprocessor from https://github.com/vrsperanza/CPY with the idea: let's make c++ look like python (but it is just some syntactical sugar). i really liked vrsperanza's idea and decided to expand it a little bit with okp: https://github.com/raisjn/okp - it's a python pre-processor for .cpy files: https://github.com/raisjn/okp/blob/master/USAGE.md

long story short: cpy files are run through okp and turned into c++ then compiled. **but** it's not necessary to use cpy to write new apps with rmkit since rmkit.h is a single file header and as such can be included from any cpp file.

**Comment by raisjnn** [comment]
I had to pull in the big guns and ask bokluk for some advice, he mentioned that this is because xochitl needs to be started with `--system` argument.

when `xochitl --system` is run, when xochitl exits, the system powers off. when it is just `xochitl`, then it is run in test mode and xochitl won't power off the system when it quits through the power menu.

i'll try it out later today, but if you want to verify, you (whoever is reading this task) can change the `xochitl` invocation to `xochitl --system` in launcher.cpy and use the power menu to turn on and off

**Comment by raisjnn** [comment]
https://github.com/rmkit-dev/rmkit/commit/b0163b91ef0644c60f539f7223f97c6f93ea3012 reads the xochitl command out of the systemd xochitl.service file (located @ /lib/systemd/system/xochitl.service).

if you update that file with the `-plugin evdevlamy`, when you run remux, you should see a line that says `XOCHITL CMD IS ...` and it should contain the `-plugin evdevlamy` portion (and remux will launch xochitl using that command)

**Comment by raisjnn** [comment]
in b0163b91ef0644c60f539f7223f97c6f93ea3012, implemented pulling xochitl command from the systemd file
